### ISSUE TITLE
Optional Start Step

### ISSUE THREAD

**Issue by CromonMS** [pulsardev/vue-tour#19]
Hello again,

I have added a simple tweak to the start() function (locally) where you can optionally add a step number and start from that step, if you omit the start step then it starts from 0 as normal:

```
start (step) {
  // Wait for the DOM to be loaded, then start the tour
  setTimeout(() => {
    this.customCallbacks.onStart()
    this.currentStep = step || 0
  }, this.customOptions.startTimeout)
},
```

A neat trick this allows is to be able to have a tooltip type effect on particular steps you desire.
```
<label class="label">Label <a @click="tourStep1"><i class="fa fa-info-circle"></i></a></label>
```

```
tourStep1 () {
  this.$tours['releaseFormTour'].start(1)
}
```
![vue-tour-tooltip-effect](https://user-images.githubusercontent.com/5735921/39269397-9bf44a8c-48ca-11e8-8d44-69111cc87cae.png)

**Comment by Outpox** [comment]
Hello again!

This is indeed something that could be added. Do you want to give it a try via a pull request?
If so you may want to read the [contributing guide](https://github.com/pulsardev/vue-tour/blob/master/CONTRIBUTING.md#commit-message-format) and checkout some of our commit to get the point of it.
Also, if I may, the variable name should be something clearer such as `initialStep` or `startStep`.

**Comment by mmorainville** [comment]
Hi @CromonMS,

This is indeed an interesting feature! I would love to see a PR for this! 😁
However please note that you can already trigger any step anytime you want by just doing this:

```
this.$tours['myTour'].currentStep = 5
```

You can do that because this.$tours['myTour'] represents the actual `VTour` component so you have access (and can change) every property of it "manually". And as the currentStep property is used internally to know which step we are on (and so which are the previous and next steps) it would just work.

**Comment by CromonMS** [comment]
Excellent, I'll take a look at this after the weekend, got a bit of stuff to deal with before then 😃

**Comment by CromonMS** [comment]
Hi,

Apologies for the delays, had a ton of stuff on!

I have made the commit on my fork here: https://github.com/CromonMS/vue-tour/commit/13d699619faf9cd1b403468245d6637b4e2dcee5

Let me know if it's okay or I need to add / change anything, haven't done much contributing to other projects yet! :)

**Comment by Outpox** [comment]
Merged in #32, thanks for your work :-)
